downstream/modules/platform/con-aap-migration-prereqs.adoc

```diff
@@ -0,0 +1,16 @@
+[id="aap-migration-prereqs"]
+
+= Prerequisites
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
```

Comment:
Suggestion: move this abstract after the [role="_abstract"] below. Otherwise, it might not display under the title as it should. (The preview doesn't display it at all.)

Reply:
Fixed.